pkg/datastore/datastore.go

```diff
@@ -367,6 +367,10 @@ type RelationshipIterator interface {
 	// Next returns the next tuple in the result set.
 	Next() *core.RelationTuple
 
+	// Cursor returns a cursor representing the last relationship returned if
```

Comment:
maybe something like:

Cursor returns a cursor that can be used to resume reading of relationships from the last relationship returned. Only applies if a sort ordering was requested.